src/components/ebay-button/index.marko

```diff
@@ -47,13 +55,20 @@ $ {
         fixedHeightClass,
         transparentClass,
         !truncateClass && !fixedHeightClass && sizeClass,
-        (priority === "secondary" ||
-            priority === "primary" ||
-            priority === "delete") &&
-            `${baseClass}--${priority}`
+        (validPriorities.includes(priority) &&
+            `${baseClass}--${priority}`)
     ]
     data-ebayui=true
     type=(tag === "button" && (input.type || "button"))
-    aria-disabled=(input.partiallyDisabled && "true")>
-    <${input.renderBody}/>
+    aria-disabled=((input.partiallyDisabled || input.loading) && "true")
+    aria-live=(input.bodyState === "loading" && "polite")
+    aria-label=(input.bodyState === "loading" && "Loading...")>
```

Comment:
We need to have a way for people to override this text for localizations. 
Sorry I just noticed this.